pkg/server/status_test.go

```diff
@@ -849,3 +852,72 @@ func TestRemoteDebugModeSetting(t *testing.T) {
 		}
 	}
 }
+
+func TestStatusAPIQueries(t *testing.T) {
```

Comment:
I guess this should be called `TestStatusAPIStatements`? (Thx for doing the rename btw)